src/client.rs

```diff
@@ -187,7 +186,11 @@ pub trait Histogrammed {
 
     /// Record a single histogram value with the given key and return a
     /// `MetricBuilder` that can be used to add tags to the metric.
-    fn histogram_with_tags<'a>(&'a self, key: &'a str, value: u64) -> MetricBuilder<'_, '_, Histogram>;
+    fn histogram_with_tags<'a>(
```

Comment:
Do you mind reverting the formatting changes here to keep the diff focused on the functionality changes?

Reply:
Done. I have `rust-fmt` attached to save, so this just got caught up in that.